src/c++/perf_analyzer/docs/cli.md

```diff
@@ -352,8 +367,8 @@ output tensor. Only needed when one or more of the outputs are of string type
 and/or variable shape. The value should be larger than the size of the largest
 output tensor that the model is expected to return. Perf Analyzer will use the
 following formula to calculate the total shared memory to allocate:
-output_shared_memory_size * number_of_outputs * batch_size.
-         
+output_shared_memory_size _ number_of_outputs _ batch_size.
```

Comment:
?

Reply:
There was an improper setting in my formatter. It should be fixed now.